src/main/java/org/bahmni/reports/report/TSIntegrationDiagnosisLineReport.java

```diff
@@ -0,0 +1,25 @@
+package org.bahmni.reports.report;
+
+
+import org.bahmni.reports.BahmniReportsProperties;
+import org.bahmni.reports.model.Report;
+import org.bahmni.reports.model.TSIntegrationDiagnosisLineReportConfig;
+import org.bahmni.reports.template.BaseReportTemplate;
+import org.bahmni.reports.template.TSIntegrationDiagnosisLineReportTemplate;
+import org.bahmni.webclients.HttpClient;
+
+
+import java.util.Properties;
+
+public class TSIntegrationDiagnosisLineReport extends Report<TSIntegrationDiagnosisLineReportConfig> implements TSHttpClient {
+
+
+    @Override
+    public BaseReportTemplate getTemplate(BahmniReportsProperties bahmniReportsProperties) {
+        HttpClient httpClient = getHttpClient(bahmniReportsProperties);
+        Properties tsProperties = getTSProperties();
+        String tsEndpointTemplate = bahmniReportsProperties.getOpenmrsRootUrl() + tsProperties.getProperty("ts.endpoint");
```

Comment:
I had left a comment in one of the discussions earlier that we don't want our Global property prefix to be "ts". It makes no sense. It should be renamed to "fhirTerminologyServer" on the UI in global properties. This is a section name and "ts" is too geeky for customers to understand what this is for.